src/app/docs/waterfall-indicator/waterfall-indicator-docs.component.html

```diff
@@ -0,0 +1,247 @@
+        </p>
+      </sky-docs-design-guideline>
+
+      <sky-docs-design-guideline
+        heading="Step details"
+      >
+        <p>
+          You can provide additional details about completed steps, but limit details to single pieces of information. Don’t go overboard because users can reopen modals to review choices as necessary.
+        </p>
+      </sky-docs-design-guideline>
+
+      <sky-docs-design-guideline
+        heading="Start over action"
+      >
+        <p>
+          You can include a start over action to let users erase all saved data from the waterfall process. This returns user to the first step or to an earlier point in the workflow if decisions occured before the waterfall progress indicator.
+        </p>
+      </sky-docs-design-guideline>
+    </sky-docs-demo-page-section>
+
+    <sky-docs-demo-page-section
+      heading="Behavior & states"
+    >
+      <sky-docs-design-guideline>
+        <p>
+          Waterfall progress indicators guide users through one active step at a time. When users complete a step, the progress indicator updates its step indicator and step action to the completed state and updates the next step indicator and step action to the active state. Users can review and edit completed steps, but this does not undo the completed state.
+        </p>
+        <p>
+          The final step summarizes the changes and requests user confirmation. You do not need to include all data that users enter in the summary, but confirmation is required. After users complete the final step, replace the waterfall progress indicator with a page that displays the finished or active state of the system.
+        </p>
+      </sky-docs-design-guideline>
+    </sky-docs-demo-page-section>
+    
+    <sky-docs-demo-page-section
+      heading="Content"
+    >
+      <sky-docs-design-guideline>
+        <p>
+          Use a waterfall title in place of an ordinary page title to provide context for the overall task. Begin the title with a verb to describe the action that the waterfall steps take when they are all completed, similar to the way modal titles are constructed.
+        </p>
+        <p>
+          Step labels in waterfall progress indicators start with a number and concisely describe the objective. Use the <b>&lt;verb> &lt;direct object></b> format. In the label for the last step, describe the cumulative change that occurs at the end of the process.
+        </p>
+        <p>
+          Step action (button) labels closely follow step labels and use the same <b>&lt;verb> &lt;direct object></b> format, but they do not need to be identical. For the first step action label, be generic and conversational (“Get started”), and for completed step actions, use verbs appropriate to post hoc actions (“Change”, “Edit”).
+        </p>
+      </sky-docs-design-guideline>
+
+      <sky-docs-design-guideline
+        heading="Layout"
+      >
+        <p>
```

Comment:
The spacing is off again, but I don't see any style applied in other phase 2 docs to enforce the narrower width.

Reply:
No idea what's going on with the spacing when serving locally. I'm going to resolve this comment, and we can create a follow-up issue later if the live docs have inconsistent spacing.